geode-docs/tools_modules/gfsh/command-pages/export.html.md.erb

```diff
@@ -246,14 +262,14 @@ Export the stack trace for a member or members.
 export stack-traces --file=value [--members=value(,value)*] [--groups=value(,value)*]
 ```
 
+**Export Stack-Traces Parameters:**
+
 | Name                                           | Description                                                     |
 |------------------------------------------------|-----------------------------------------------------------------|
 | <span class="keyword parmname">\\-\\-file</span>   | *Required.* Filename to which the stack-traces will be written. |
 | <span class="keyword parmname">\\-\\-members</span> | Name or ID of the member(s) whose log files will be exported.      |
 | <span class="keyword parmname">\\-\\-groups</span>  | Group(s) of members whose log files will be exported.          |
 
-<span class="tablecap">Table 6. Export Stack-Traces Parameters</span>
-
 **Example Commands:**
 
 ``` pre
```

Comment:
While making other changes, can you update the gfsh export data Sample Output subsection?  It'd be nice to not implicitly refer to GemFire 7.0 in the file path.